doc/faq.rst

```diff
@@ -0,0 +1,54 @@
+Frequently asked questions
+==========================
+
+Should I apply ICA first or autoreject first?
+---------------------------------------------
+
+ICA solutions can be affected by high amplitude artifacts, therefore
+we recommend first using autoreject to detect the bad segments, then applying
+ICA, and finally interpolating the bad data.
+
+To ignore bad segments using autoreject (local), we could do::
+
+	>>> ar = Autoreject()
+	>>> _, reject_log = ar.fit(epochs).transform(epochs)
+	>>> ica.fit(epochs[~reject_log.bad_epochs_idx])
+
+or use autoreject (global)::
+
+	>>> reject = get_rejection_threshold(epochs)
+	>>> ica.fit(epochs, reject=reject)
+
```

Comment:
Perhaps add a note here that this strategy would work easily when fitting ICA on raw data while the other one needs epochs.